renderdoc/driver/shaders/spirv/spirv_debug_setup.cpp

```diff
@@ -966,6 +966,15 @@ ShaderDebugTrace *Debugger::BeginDebug(DebugAPIWrapper *api, const ShaderStage s
           memset(&var.value, 0xcc, sizeof(var.value));
         }
 
+        // VK_KHR_maintenance5 requires that if PointSize is not written during shader execution, it should be 1.0f
+        //   so we will initialize it to 1.0f here (if it is not enabled then leaving it as 1.0f is harmless)
+        if(!isInput && curDecorations.flags & Decorations::HasBuiltIn &&
+           curDecorations.builtIn == BuiltIn::PointSize)
+        {
+          RDCASSERTEQUAL(var.type, VarType::Float);
+          var.value.f32v[0] = 1.0f;
+        }
```

Comment:
What's the motivation for this change? The behaviour difference in `VK_KHR_maintenance5` is effectively considered _after_ the shader has executed, if `PointSize` is not written to. I don't think that means we should pretend that it's initialised to that value as that feels misleading.

Reply:
OK, will revert

Reply:
I do think it's not necessary but I would like to know the reason you had for adding the code originally. Maybe it's something I hadn't considered or maybe you're trying to solve a problem which needs to be solved a different way.

Reply:
My train of thought was that I should mark that the pointSize has such behaviour. But now that I am thinking about it if someone is seeing that PointSize turns out to be 1. (as side effects) even if he hadn't written to it is better than having to debug if this ext is not enabled why PointSize is shown in debugger to be 1. but they are rendered with garbage PointSize.